pom.xml

```diff
@@ -71,28 +71,32 @@
     <dependency>
       <groupId>org.cactoos</groupId>
       <artifactId>cactoos</artifactId>
-      <version>0.44</version>
+      <version>0.45</version>
       <scope>provided</scope>
     </dependency>
+    <!--
+     @todo #1411:30min Migrate existing tests to JUnit 5. It mainly means
+      changing the imports and move from Ignore to Disabled annotation as
+      well as replace rules with corresponding annotations. Once this is done
+      remove the dependencies below
```

Comment:
@victornoel which dependencies? I think it better to be clearer here.

Reply:
@fabriciofx thx it is done